packages/components/addon/components/hds/button/index.ts

```diff
@@ -5,7 +5,7 @@
 
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
-import { HdsInteractiveSignature } from '../interactive';
+import { type HdsInteractiveSignature } from '../interactive';
```

Comment:
[question] this is technically a user facing change that would require a changeset, yes? If so, can we pull it out into a separate PR or pull it into one of the ones that already has user facing changes?

Reply:
ah, this little one made its way in – good catch! it's enforced by the tsconfig update. will pull it out